src/emulator/commandUtils.ts

```diff
@@ -335,6 +335,13 @@ async function runScript(script: string, extraEnv: Record<string, string>): Prom
     env[FirestoreEmulator.FIRESTORE_EMULATOR_ENV_ALT] = address;
   }
 
+  const authInstance = EmulatorRegistry.get(Emulators.AUTH);
+  if (authInstance) {
+    const info = authInstance.getInfo();
+    const address = `${info.host}:${info.port}`;
```

Comment:
I really hate to be that person, bit please quote IPv6 addresses

Reply:
@yuchenshi I did it everywhere ... hope that wasn't a bad idea!